File: MODUL 3 C/PRAK205-2210817310004-Muhammad Adika Rizqullah.c
#include <stdio.h>

int main(){
float A, B, C, Keliling, Luas;

scanf("%f %f", &A, &B);

C = sqrt(B*B-A*A);
Keliling = A+B+C;
Luas = 0.5*C*A;

printf("Alas = %.0f cm\n", C);
printf("Tinggi = %.0f cm\n", A);
printf("Keliling = %.0f cm\n", Keliling);
printf("Luas = %.0f cm^2\n", Luas);

}

File: MODUL 3 PYTHON/PRAK205-2210817310004-Muhammad Adika Rizqullah.py
import math
tinggi, miring = input().split()
tinggi = float(tinggi)
miring = float(miring)
alas = math.sqrt(miring*miring-tinggi*tinggi)
keliling = tinggi + miring + alas
luas = 0.5*alas*tinggi
print("Alas =", round(alas), "cm")
print("Tinggi =", round(tinggi), "cm")
print("Keliling", round(keliling), "cm")
print("Luas =", round(luas), "cm^2\n")

bTinggi = input()
bMiring = input()
bTinggi = float(bTinggi)
bMiring = float(bMiring)
bAlas = math.sqrt(bMiring*bMiring-bTinggi+bTinggi)
bKeliling = bTinggi+bMiring+bAlas
bLuas = 0.5*bAlas*bTinggi
print("Alas =", round(bAlas), "cm")
print("Tinggi =", round(bTinggi), "cm")
print("Keliling =", round(bKeliling), "cm")
print("Luas =", round(bLuas), "cm^2")